Dockerfile

```diff
@@ -140,7 +142,7 @@ RUN --mount=from=containerd-src,src=/usr/src/containerd,readwrite --mount=target
 # containerd v1.5 for integration tests
 FROM containerd-base as containerd-alt-15
 ARG CONTAINERD_ALT_VERSION_15
-ARG GO111MODULE=off
+ENV GO111MODULE=off
```

Comment:
curious; What was the reason for this change?

Reply:
It shouldn't be a configurable value because no other value works in here. Could also just be inlined in the build command.

Reply:
Actually, why is this set at all? 1.5 is already using go.mod so should not need it but 1.4 also seems to build fine without it.

Reply:
I think it may have been from before go modules did vendor detection

Or this was related to the `//import` comments otherwise not being detected 🤔

As to "why an ARG", I guess because it's sometimes more convenient than exporting or inlining, and not always something that's needed in the image itself (similar to DEBIAN_FRONTEND vars)